Dockerfile

```diff
@@ -1,3 +1,15 @@
+FROM docker.io/library/alpine:3.14 as restic
+
+RUN apk add --update --no-cache \
+    bash \
+    ca-certificates \
+    curl
+
+ARG RESTIC_VERSION=0.12.1
+COPY fetch_restic.sh ./
```

Comment:
What do you think about the idea where the restic version is managed via go.mod/tools.go as a blind import so that we get Renovate support? We could then parse the version similar to the GitHub action in the "setup-go" step from the go.mod file instead.